internal/load/buildrun.go

```diff
@@ -146,9 +146,15 @@ func ExecuteSingleBuildRun(kubeAccess KubeAccess, namespace string, name string,
 		return nil, err
 	}
 
+	// Force clean env before starting test.
+	debug("Force clean env before starting test")
+
+	var graceperiod int64 = int64(0)
+	podDeleteOptions := metav1.DeleteOptions{GracePeriodSeconds: &graceperiod}
```

Comment:
Just to avoid confusion since it is not a pod, it should maybe called `deleteOptions` instead.
```suggestion
	deleteOptions := metav1.DeleteOptions{GracePeriodSeconds: &graceperiod}
```

Reply:
Done.